src/module-1/cutStrings/index.js

```diff
@@ -1 +1,11 @@
-export const cutStrings = (arr = []) => {};
+export const cutStrings = (arr = []) => {
+   const result = arr.map(word => {
+      return word.length
+   })
+   let minElm = Math.min(...result)
+   const results = arr.map(word2 => {
```

Comment:
try to don't use the numbers in variables declaration
can be `word` like in 2 line